src/structures/StickerPack.js

```diff
@@ -10,10 +10,6 @@ const SnowflakeUtil = require('../util/SnowflakeUtil');
  * @extends {Base}
  */
 class StickerPack extends Base {
-  /**
-   * @param {Client} client The instantiating client
-   * @param {APIStickerPack} pack The data for the sticker pack
```

Comment:
This was the last instance of `APIStickerPack`. Should we remove this too?

https://github.com/discordjs/discord.js/blob/395186da7241b1ea704e4d5e0c6df6fd4655eb5c/src/structures/StickerPack.js#L97-L100